docs/index.yaml

```
@@ -2,7 +2,16 @@ apiVersion: v1
 entries:
   fahclient:
   - apiVersion: v2
-    created: "2021-01-27T20:55:21.071586+01:00"
+    created: "2021-05-15T18:09:43.510132376+02:00"
+    description: A resilient Helm chart for Folding@Home with GPU support
+    digest: a66bd563a5b973cdbe6c08ce27fa1fec6ad0ded1a043222788d06993987e9471
+    name: fahclient
+    type: application
+    urls:
+    - https://pcktdmp.github.io/charts/fahclient-2.6.5.tgz
+    version: 2.6.5
```

```suggestion
    version: 2.6.0
```
And the corresponding changes elsewhere.